core/templates/pages/story-editor-page/services/story-editor-state.service.ts

```diff
@@ -206,25 +210,29 @@ export class StoryEditorStateService {
       return false;
     }
     this._storyIsBeingSaved = true;
-    this.editableStoryBackendApiService.updateStoryAsync(
-      this._story.getId(), this._story.getVersion(), commitMessage,
-      this.undoRedoService.getCommittableChangeList() as StoryChange[]
-    ).then(
-      (storyBackendObject) => {
-        this._updateStory(storyBackendObject);
-        this.undoRedoService.clearChanges();
-        this._storyIsBeingSaved = false;
-        if (successCallback) {
-          successCallback();
-        }
-      }, error => {
-        let errorMessage = error || 'There was an error when saving the story.';
-        this.alertsService.addWarning(errorMessage);
-        this._storyIsBeingSaved = false;
-        if (errorCallback) {
-          errorCallback(errorMessage);
-        }
-      });
+    const storyId = this._story.getId();
+    if (storyId !== null) {
```

Comment:
Should we throw an error if `storyId` is `null`?

Reply:
ditto (handler problem)

Reply:
ditto

Reply:
Removed